### DISCUSSION TITLE
Amazon Thermostat?

### DISCUSSION THREAD

**Discussion by ghubg10001** [dgreif/ring#1058] [Q&A]
I have used the Ring plugin with a lot of success. It is really great how I can arm/disarm my security system, see cameras, or control several other features in HomeKit with HomeBridge. 

**My question:** I was looking to get the Amazon Smart Thermostat. It is $60 which is a great deal. I would like being able to control the temp from the Ring app, which I use a lot, Alexa, OR in HomeKit. But does the thermostat port into HomeKit as easily as the cameras, sensors, security, and everything else with the HomeBridge plugin? Has anyone done this?

**Comment by dgreif** [comment]
I'm not aware of anyone trying to integrate a thermostat via the homebridge plugin yet. I'm sure it has Alexa support if it's from Amazon
